### ISSUE TITLE
Warnings are a bit annoying if libzim is installed system-wide

### ISSUE THREAD

**Issue by legoktm** [openzim/python-libzim#70]
I'm working on the Debian packaging for python-libzim. It builds against `libzim-dev`, which is installed to `/usr/include/zim`. The build process correctly finds it and builds properly (yay!) but it annoyingly constantly puts out warnings like:

```
[!] Warning: Couldn't find zim/*.h in ./include!
    Hint: You can install them from source from https://github.com/openzim/libzim
          or download a prebuilt release's headers into ./include/zim/*.h
          (or set CFLAGS='-I<library_path>/include')
[!] Warning: Couldn't find libzim.so in ./lib or system library paths!    Hint: You can install it from source from https://github.com/openzim/libzim
          or download a prebuilt libzim.so release into ./lib.
          (or set LDFLAGS='-L<library_path>/lib/[x86_64-linux-gnu]')
```

Could we suppress the warnings if CFLAGS/LDFLAGS are set in the environment?

**Comment by kelson42** [comment]
@rgaudin @legoktm IMO it should just work even without explict CFLAGS as the headers are in the default system place.

**Comment by mgautierfr** [comment]
This is working. It is just that we print a warning if the headers/library are not in the local directory tree.

**Comment by legoktm** [comment]
Yep, everything builds fine, it's just that the warnings are annoying because there's nothing actionable about them.

**Comment by kelson42** [comment]
@rgaudin @mgautierfr I don't understand the warning either here. I think it would be better to:
* have an error if libzim headers and/or binaries are not available with the hint already given (so same as now, but "error" level).
* We need an info message (so not a warning) telling us against which (path) header/binaries the binding is compiling otherwise.
